zk/stages/stage_sequence_execute_blocks.go

```diff
@@ -191,7 +190,6 @@ func finaliseBlock(
 	finalHeader := finalBlock.HeaderNoCopy()
 	finalHeader.Root = newRoot
 	finalHeader.Coinbase = batchState.getCoinbase(batchContext.cfg)
-	finalHeader.GasLimit = utils.GetBlockGasLimitForFork(batchState.forkId)
```

Comment:
Not sure why this was missed, but it is a required change. We don't want to reset gas limit when finalizing block because it will break non-hermez chains.

Reply:
But it will break hermez chains as it affects the block hash so we need to keep this in

Reply:
Let's see what breaks when it goes in and just fix forwards